examples/prompting/Basic_Reasoning.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    math_problem = template.format(problem="Given a triangle with base 𝑏=6 and height h=8, calculate its area")
</pre><p> </p><p>Nit: Suggest using either regular ascii or unicode characters for math symbols, don't mix them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
With the recent push I think the mix of characters has been updated -- but let's double check once more how it appears.